.github/workflows/ci_workflows.yml

```diff
@@ -43,26 +43,16 @@ jobs:
           #  toxenv: py37-test-oldestdeps
 
           # APPEND -cov TO toxenv FOR COVERAGE
-          - name: Linux - Python 3.9
+          - name: Linux - Python 3.7
             os: ubuntu-latest
-            python: 3.9
-            toxenv: py39-test
+            python: 3.7
+            toxenv: py37-test
 
           - name: OS X - Python 3.7
             os: macos-latest
             python: 3.7
             toxenv: py37-test
 
-          - name: Windows - Python 3.8
-            os: windows-latest
-            python: 3.8
-            toxenv: py38-test
-
-          - name: Python 3.9 with latest dev versions of key dependencies
-            os: ubuntu-latest
-            python: 3.9
-            toxenv: py39-test-devdeps
```

Comment:
Change this to Python 3.8 because you need tensorflow, as you said. But I'd rather keep the jobs if they are not failing.